packages/web/src/components/track/LockedStatusBadge.tsx

```diff
@@ -5,14 +5,22 @@ import styles from './LockedStatusBadge.module.css'
 
 export type LockedStatusBadgeProps = {
   locked: boolean
+  variant: 'premium' | 'gated'
```

Comment:
naming a bit confusing since we use premium content and gated content somewhat interchangeably prior to usdc. would it make more sense to just use `usdc` and `gated`?

Reply:
Thanks for context, @rickyrombo !
My thinking was indeed that we are making a shift to referring to purchasable content as "premium". But we do have a lot of lingering instances of "premium" referring to gated content in general.
This was my attempt to start using the new terminology.

Elsewhere in this PR I think I also mixed in the word "purchased" as an indication of usdc-gated content. So I at least need to pick one or the other going forward.

I think it probably deserves a discussion as a team on whether we want to actually make the change in code, or just leave a mapping between code->display strings and know that we refer to "premium" content internally as "purchasable" content.